.ruby-gemset

```diff
@@ -0,0 +1 @@
+heartcheck
```

Comment:
I think:

- having ```export rvm_ignore_gemsets_flag=1``` in a dotfile, the  problem solved globally
- but if a need to ```echo 'projectname' > .ruby-gemset``` in each project, is much work